nixos/modules/misc/ids.nix

```diff
@@ -352,6 +352,7 @@ in
       moonraker = 320;
       distcc = 321;
       webdav = 322;
+      asf = 323;
```

Comment:
See https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md as to why you should not do this. Use `users.users.asf.isSystemUser = true;` instead.